meta-imx8mmsolidrun-support/conf/machine/imx8mmsolidrun.conf

```diff
@@ -16,6 +16,9 @@ MACHINE_FEATURES += "pci wifi bluetooth optee"
 UBOOT_MACHINE_imx8mmsolidrun = "imx8mm_solidrun_defconfig"
 SPL_BINARY = "spl/u-boot-spl.bin"
 
+ATF_PLATFORM = "imx8mm"
```

Comment:
So, they've not managed to do the rename "sort of completely", but left it there sort of half-way? Interesting.